stl/inc/yvals_core.h

```diff
@@ -381,19 +381,6 @@
 
 #include <vcruntime.h>
 

```

Comment:
The formatting gods want that line here to be gone.

Note that this should be safe because `vcruntime` is lexicographically first

Reply:
Fixed, thanks. For future reference, is there any way to run the code format validation check locally?